src/partner/ducks/eventOrders.ts

```diff
@@ -0,0 +1,180 @@
+      return { ...state, unpaidOrders };
+    case EVENT_ORDERS_CHECK_ONE_PAID:
+      let newPaidOrdersChecked = state.paidOrders.map((order: IOrder) => {
+        if (order.id === action.payload.orderId) {
+          order.checked = action.payload.checked;
+        }
+        return order;
+      });
+      return { ...state, paidOrders: newPaidOrdersChecked };
+    case EVENT_ORDERS_CHECK_ONE_UNPAID:
+      let newUnpaidOrdersChecked = state.unpaidOrders.map((order: IOrder) => {
+        if (order.id === action.payload.orderId) {
+          order.checked = action.payload.checked;
+        }
+        return order;
+      });
+      return { ...state, unpaidOrders: newUnpaidOrdersChecked };
+    case MOVE_CHECKED_ORDERS_TO_PAID:
+      const checkedUnpaidOrders = state.unpaidOrders.filter((order: IOrder) => order.checked);
+      const uncheckedUnpaidOrders = state.unpaidOrders.filter((order: IOrder) => !order.checked);
+      return {
+        ...state,
+        paidOrders: [...state.paidOrders, ...checkedUnpaidOrders],
+        unpaidOrders: uncheckedUnpaidOrders
+      };
+    case MOVE_CHECKED_ORDERS_TO_UNPAID:
+      const checkedPaidOrders = state.paidOrders.filter((order: IOrder) => order.checked);
+      const uncheckedPaidOrders = state.paidOrders.filter((order: IOrder) => !order.checked);
+      return {
+        ...state,
+        unpaidOrders: [...state.unpaidOrders, ...checkedPaidOrders],
+        paidOrders: [...uncheckedPaidOrders]
+      };
+    default:
+      return state;
+  }
+}
+
+export const fetchEventOrdersStarted = () => {
+  return { type: FETCH_EVENT_ORDERS_START };
+};
+
+export const fetchEventOrdersSucceed = (event: IOrder[]) => {
+  return { type: FETCH_EVENT_ORDERS_SUCCESS, payload: event };
+};
+
+export const fetchEventOrdersFailed = (error: Error) => {
+  return { type: FETCH_EVENT_ORDERS_FAIL, payload: error };
+};
+
+export const checkAllPaidOrders = (checked: boolean) => {
+  return { type: EVENT_ORDERS_CHECK_ALL_PAID, payload: checked };
+};
+
+export const checkAllUnpaidOrders = (checked: boolean) => {
+  return { type: EVENT_ORDERS_CHECK_ALL_UNPAID, payload: checked };
+};
+
+export const checkPaidOrderById = (orderId: string, checked: boolean) => {
+  return { type: EVENT_ORDERS_CHECK_ONE_PAID, payload: { orderId, checked } };
+};
+
+export const checkUnpaidOrderById = (orderId: string, checked: boolean) => {
+  return { type: EVENT_ORDERS_CHECK_ONE_UNPAID, payload: { orderId, checked } };
+};
+
+export const moveCheckedOrdersToPaid = () => {
+  return { type: MOVE_CHECKED_ORDERS_TO_PAID };
+};
+
+export const moveCheckedOrdersToUnpaid = () => {
+  return { type: MOVE_CHECKED_ORDERS_TO_UNPAID };
+};
+
+export const moveOrders = (paidTab: boolean, dispatch: dispatchType) => {
+  if (paidTab) {
+    dispatch(moveCheckedOrdersToUnpaid());
+  } else {
+    dispatch(moveCheckedOrdersToPaid());
+  }
+};
+
+export const checkAllOrders = (paidOrders: boolean, checked: boolean, dispatch: dispatchType) => {
```

Comment:
this function shouldn't be here, it should located at `EventOrdersContainer.tsx`